### PR TITLE
[TACHYON-1215] rmr doesn't show error message

### PR DESCRIPTION
**Pull request by leohjwang**
https://tachyon.atlassian.net/browse/TACHYON-1215

### CODE REVIEW

File: shell/src/main/java/tachyon/shell/command/RmrCommand.java [hunk 1]
```diff
@@ -46,6 +47,8 @@ void runCommand(TachyonURI path) throws IOException {
       TachyonFile fd = mTfs.open(path);
       mTfs.delete(fd, options);
       System.out.println(path + " has been removed");
+    } catch (InvalidPathException e) {
+      System.out.println(path + " does not exists");
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: exists -> exist

**Comment by aaudiber** [comment]
@leohjwang Thanks for the PR! Please add a link to the JIRA ticket at the start of the PR description. I left a comment on the code.

File: shell/src/main/java/tachyon/shell/command/RmrCommand.java [hunk 2]
```diff
@@ -46,6 +47,8 @@ void runCommand(TachyonURI path) throws IOException {
       TachyonFile fd = mTfs.open(path);
       mTfs.delete(fd, options);
       System.out.println(path + " has been removed");
+    } catch (InvalidPathException e) {
+      System.out.println(path + " does not exist");
```
Comment (calvinjia) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: I think the error message should be `is an invalid path`? I think `FileDoesNotExistException` will be thrown for the does not exist case.

Comment (leohjwang) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: Currently AbstractTachyonFileSystem.open() throws an InvalidPathException when the file does not exist. That's why we need to catch this exception here.

Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: What about `println(e.getMessage())`?

**Comment by apc999** [comment]
@leohjwang 
This PR is close to be merged. If you have time, could you address the comments on this PR? Thanks

File: shell/src/main/java/tachyon/shell/command/RmrCommand.java (hunk 2 above)
Comment (leohjwang) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: @uronce-cc I don't think it's a good idea to print out the exception in tfs shell. We should make the tfs shell message to be as much concise and clean as possible, and currently other commands do not print out exception messages as well.

**Comment by apc999** [comment]
+1 LGTM

**Comment by uronce-cc** [comment]
+1 LGTM

**Pull request merged**